tutorials/triggered_publisher.md

```diff
@@ -0,0 +1,255 @@
+\page triggeredpublisher Triggered Publisher
+
+The `TriggeredPublisher` system publishes a user specified message on an output
+topic in response to an input message that matches user specified criteria. The
+system works by checking the input against a set of Matchers. Matchers
+contain string representations of protobuf messages which are compared for
+equality or containment with the input message. Matchers can match the whole
+input message or only a specific field inside the message.
+
+This tutorial describes how the Triggered Publisher system can be used to
+cause a box to fall from its initial position by detaching a detachable joint
+in response to the motion of a vehicle. The tutorial also covers how Triggered
+Publisher systems can be chained together by showing how the falling of the box
+can trigger another box to fall. The finished world SDFormat file for this
+tutorial can be found in 
+[examples/worlds/triggered_publisher.sdf](https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo2/examples/worlds/triggered_publisher.sdf)
+
+We will use the differential drive vehicle from
+[examples/worlds/diff_drive.sdf](https://github.com/ignitionrobotics/ign-gazebo/blob/ign-gazebo2/examples/worlds/diff_drive.sdf), 
+but modify the input topic of the `DiffDrive` system to `cmd_vel`. A snippet of 
+the change to the `DiffDrive` system is shown below:
+
+\code{.xml} 
+  <model name='vehicle_blue'>
+    ...
+
+    <plugin
+      filename="libignition-gazebo-diff-drive-system.so"
+      name="ignition::gazebo::systems::DiffDrive">
+      <left_joint>front_left_wheel_joint</left_joint>
+      <left_joint>rear_left_wheel_joint</left_joint>
+      <right_joint>front_right_wheel_joint</right_joint>
+      <right_joint>rear_right_wheel_joint</right_joint>
+      <wheel_separation>1.25</wheel_separation>
+      <wheel_radius>0.3</wheel_radius>
+      <topic>cmd_vel</topic>
+    </plugin>
+  </model>
+\endcode
+
+The first `TriggeredPublisher` we create will demonstrate how we can send
+a predetermined `Twist` message to the `DiffDrive` vehicle in response to
+a "start" message from the user:
+
+\code{.xml}
+<plugin filename="libignition-gazebo-triggered-publisher-system.so" name="ignition::gazebo::systems::TriggeredPublisher">
+  <input type="ignition.msgs.Empty" topic="/start"/>
+  <output type="ignition.msgs.Twist" topic="/cmd_vel">
+      linear: {x: 3}
+  </output>
+</plugin>
+\endcode
```

Comment:
this might be a feature of our docs, but just pointing out that the only thing rendering here in github's preview is `linear: {x: 3}` in case it's not supposed to be like that

Reply:
Good catch. I switched the code blocks to markdown fenced code blocks. Renders much better now. https://github.com/ignitionrobotics/ign-gazebo/pull/139/commits/2ac8ebf93ad37f1e190c94ec89c27b7208fc42ae